php.ini-development

```diff
@@ -537,6 +537,7 @@ ignore_repeated_errors = Off
 ; https://php.net/ignore-repeated-source
 ignore_repeated_source = Off
 
+; DEPRECATED: This INI setting is deprecated and will be removed in PHP 9.0.
```

Comment:
This formatting is inconsistent with the other deprecated options, please unify.

I also wonder if there still is value in having this option readily presented in the example INI files. It's deprecated and only affects a debug build. Perhaps we can just remove this entire block for improved DX? If we want to keep it, the INI should be commented, though (to make sure it uses the built-in default).